tests/Unit/Configuration/Definition/ActionConfigDefinitionTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorCommon\Tests\Unit\Configuration\Definition;
+
+use Keboola\DbExtractorCommon\Configuration\BaseExtractorConfig;
+use Keboola\DbExtractorCommon\Configuration\Definition\ActionConfigDefinition;
+use Keboola\DbExtractorCommon\Tests\Unit\Configuration\__fixtures\ConfigParametersProvider;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+
+class ActionConfigDefinitionTest extends TestCase
+{
+    public function testConfigDatabaseNodeEmptyThrowsException(): void
+    {
+        $configuration = [
+            'parameters' => [
+                'db' => [],
+            ],
+        ];
+
+        $this->expectException(InvalidConfigurationException::class);
+        $this->expectExceptionMessage('The child node "user" at path "root.parameters.db" must be configured.');
```

Comment:
just fyi, I think these can be combined into a single expression using `$this->setExpectedException(class, message)`

Reply:
I saw you are using it and seems to be handy but it was removed in PHPUnit 6.